runners/flink/pom.xml

```diff
@@ -40,6 +40,14 @@
       <activation><activeByDefault>false</activeByDefault></activation>
       <build>
         <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-compiler-plugin</artifactId>
+            <configuration>
+              <source>1.8</source>
```

Comment:
For some reason, compilation fails on my machine even with Java 8 set for the `runners/flink` module. I'm reverting it for now and adding a TODO to refactor it once Java 8 is the default.